jvm/src/test/java/com/microsoft/bond/protocol/StreamHelperTest.java

```diff
@@ -0,0 +1,99 @@
+package com.microsoft.bond.protocol;
+
+import com.sun.xml.internal.ws.util.StreamUtils;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.EOFException;
+import java.io.IOException;
+import java.util.Arrays;
+
+import static org.junit.Assert.*;
+
+public class StreamHelperTest {
+
+    @Test
+    public void callImplicitDefaultConstructor() {
+        // adds the implicit default constructor to unit test code coverage
+        new StreamHelper();
```

Comment:
Made them private and removed the dummy UT's.

---
In reply to: [116857962](https://github.com/tstein/bond/pull/4#discussion_r116857962) [](ancestors = 116857962)